VideoLocker/res/values/strings.xml

```diff
@@ -260,22 +260,19 @@
     <string name="discussion_priviledged_author_label_staff">Staff</string>
     <!-- The label to use when indicating a post was authored by a privileged user -->
     <string name="discussion_priviledged_author_attribution">By %s</string>
-    <!-- The date and author of a discussion comment, question, or thread, e.g. "3 days ago by Brian (Staff)" -->
-    <string name="post_attribution">{time} by {author} ({author_label})</string>
-    <!-- The date and author of a discussion comment, question, or thread without the author's label, e.g. "3 days ago by Brian" -->
-    <string name="post_attribution_without_label">{time} by {author}</string>
-    <!-- The date and author of an answer to a discussion question e.g. "Marked as answer 2 days ago by Khalid (Community TA)" -->
-    <string name="answer_author_attribution">Marked as answer {time} by {author} ({author_label})</string>
-    <!-- The date and author of an answer to a discussion question without the author's label e.g. "Marked as answer 2 days ago by Khalid" -->
-    <string name="answer_author_attribution_without_label">Marked as answer {time} by {author}</string>
-    <!-- The date and author of an endorsed response to a discussion thread e.g. "Endorsed 2 days ago by Khalid (Community TA)" -->
-    <string name="endorser_attribution">Endorsed {time} by {author} ({author_label})</string>
-    <!-- The date and author of an endorsed response to a discussion thread without the author's label e.g. "Endorsed 2 days ago by Khalid" -->
-    <string name="endorser_attribution_without_label">Endorsed {time} by {author}</string>
     <!-- Label for the visibility of a discussion post that is only visible to cohort -->
     <string name="discussion_post_visibility_cohort">This post is visible only to {cohort}.</string>
     <!-- Label for the visibility of a discussion post that is visible to everyone -->
     <string name="discussion_post_visibility_everyone">This post is visible to everyone.</string>
+    <!-- Used while constructing the author attribution text for a discussion post e.g. "3 days ago by Brian (Staff)" -->
+    <string name="author_attribution_prefix">by {author}</string>
+    <!-- Used while constructing the author attribution text for an answer response e.g. "Marked as answer 2 days ago by Khalid (Community TA)" -->
+    <string name="marked_as_answer_prefix">Marked as answer</string>
+    <!-- Used while constructing the author attribution text for an endorsed response e.g. "Endorsed 2 days ago by Khalid" -->
+    <string name="endorsed_prefix">@string/discussion_responses_endorsed</string>
+    <!-- Used for putting the given text into parenthesis e.g. (Community TA) -->
+    <string name="text_in_parenthesis">({text})</string>
```

Comment:
>In any case, maybe add some description in it's label, since this is the only place it's being used?

see the description above the string's declaration:
```xml
<!-- Used for putting the given text into parenthesis e.g. (Community TA) -->
<string name="text_in_parenthesis">({text})</string>
```

>Also, all of these strings should have the discussion_post_ prefix in their labels I guess.

ok.